### PR TITLE
Use bnd plugin to add OSGi manifest info

### PR DESCRIPTION
**Pull request by tdraier**
Allows to deploy jar in OSGi container without wrapping it

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,6 +17,9 @@ apply plugin: 'maven'
 apply plugin: 'maven-publish'
 apply plugin: 'antlr'
 
+apply plugin: 'osgi'
```
Comment (andimarek) [inline build.gradle]: @tdraier Thank you. Could you add a short comment here what both plugins actually do?

Comment (tdraier) [inline build.gradle]: Good question .. honestly, I took the configuration from graphql-java-annotations and graphql-java-servlet which is working well. Usually you should choose between the 2 plugins - both actually extend the manifest and the jar with in a different way. But in our case osgi plugin is enough, and it's more clear to use only one. I will remove the bnd plugin. Thank you !

Comment (bbakerman) [inline build.gradle]: This puts OSGi manifest instructions in place for those that use OSGi based systems

In this case the jar gets

```
viewmf build/libs/graphql-java-2017-06-29T10-48-10.jar
Manifest-Version: 1.0
Bnd-LastModified: 1498697292000
Bundle-ManifestVersion: 2
Bundle-Name: graphql-java
Bundle-SymbolicName: com.graphql-java
Bundle-Version: 2017.0.0.06-29T10-48-10
Created-By: 1.8.0_71 (Oracle Corporation)
Export-Package:
    graphql
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql.execution
            graphql.execution.instrumentation
            graphql.language
            graphql.schema
    graphql.execution
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.execution.instrumentation
            graphql.language
            graphql.schema
    graphql.execution.batched
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.execution
            graphql.schema
    graphql.execution.instrumentation
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.execution.instrumentation.parameters
            graphql.language
            graphql.validation
    graphql.execution.instrumentation.parameters
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.execution
            graphql.language
            graphql.schema
    graphql.introspection
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.language
            graphql.schema
    graphql.language
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.schema
    graphql.parser
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.language
            graphql.parser.antlr
    graphql.parser.antlr
        version="2017.0.0.06-29T10-48-10"
        uses:=
            org.antlr.v4.runtime
            org.antlr.v4.runtime.atn
            org.antlr.v4.runtime.dfa
            org.antlr.v4.runtime.tree
    graphql.relay
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql.schema
    graphql.schema
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.execution
            graphql.introspection
            graphql.language
    graphql.schema.idl
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.language
            graphql.schema
            graphql.schema.idl.errors
    graphql.schema.idl.errors
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.language
    graphql.schema.validation
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.schema
    graphql.validation
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql
            graphql.language
            graphql.schema
    graphql.validation.rules
        version="2017.0.0.06-29T10-48-10"
        uses:=
            graphql.language
            graphql.schema
            graphql.validation
Import-Package:
    graphql
        version="[2017.0,2018)"
    graphql.execution
        version="[2017.0,2018)"
    graphql.execution.instrumentation
        version="[2017.0,2018)"
    graphql.execution.instrumentation.parameters
        version="[2017.0,2018)"
    graphql.introspection
        version="[2017.0,2018)"
    graphql.language
        version="[2017.0,2018)"
    graphql.parser
        version="[2017.0,2018)"
    graphql.parser.antlr
        version="[2017.0,2018)"
    graphql.schema
        version="[2017.0,2018)"
    graphql.schema.idl.errors
        version="[2017.0,2018)"
    graphql.schema.validation
        version="[2017.0,2018)"
    graphql.validation
        version="[2017.0,2018)"
    graphql.validation.rules
        version="[2017.0,2018)"
    org.antlr.v4.runtime
        version="[4.5,5)"
    org.antlr.v4.runtime.atn
        version="[4.5,5)"
    org.antlr.v4.runtime.dfa
        version="[4.5,5)"
    org.antlr.v4.runtime.misc
        version="[4.5,5)"
    org.antlr.v4.runtime.tree
        version="[4.5,5)"
    org.slf4j
        version="[1.7,2)"
Require-Capability:
    osgi.ee
        filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Tool: Bnd-3.2.0.201605172007
```

ps. that viewmf tool is a homegrown tool - not a public tool

Comment (tdraier) [inline build.gradle]: Actually, that's exactly what I get with the osgi plugin only, without biz.aQute.bnd.builder (only in my case the version of bnd tool differs when I include biz.aQute.bnd.builder,  it's Bnd-3.1.0.201512181341). So including it or not does not change anything, better keep osgi plugin only.

**Comment by andimarek** [comment]
@tdraier  thanks!

**Pull request merged**